prime-router/src/testIntegration/resources/datatests/HL7_to_FHIR/sample_co_1_20220518-0001.hl7

```diff
@@ -2,6 +2,8 @@ MSH|^~\&#|MEDITECH^2.16.840.1.114222.4.3.2.2.1.321.111^ISO|COCAA^1.2.3.4.5^ISO|D
 SFT|MEDITECH, Inc.^L^^^^MEDITECH&1.3.6.1.4.1.24310&ISO^XX^^^MEDITECH|5.67|Laboratory Applicati|20210810||20210810
```

Comment:
Do not update this file.  This is the original HL7.

Reply:
reverted